whylog/tests/tests_config/__init__.py

```diff
@@ -135,16 +139,13 @@ def test_parser_serialization(self):
         assert dumped_parsers_again == dumped_parsers
 
     def test_loading_single_rule_its_parsers(self):
-        #TODO: modify test if reviewers accept UserParserIntent changes
-        raise SkipTest
-        path = os.path.join(*path_test_files)
-        parsers_path = os.path.join(path, 'parsers.yaml')
-        rules_path = os.path.join(path, 'rules.yaml')
-        log_type_path = os.path.join(path, 'log_types.yaml')
-
-        config = YamlConfig(parsers_path, rules_path, log_type_path)
-        assert len(config._rules) == 1
-        rule = config._rules[0]
+        rule = self.config._rules['lostdata'][0]
         assert sorted([cause.name for cause in rule._causes] + [rule._effect.name]) == sorted(
-            parser.name for parser in config._parsers.values()
+            parser.name for parser in self.config._parsers.values()
         )
+
+    def test_loading_log_types(self):
```

Comment:
IMO the docstring describing what does this test really tests should be written here. It's not obvious, becase this test starts with asserts and someone who call this test should guarantee that evironment in this class was prepared correclty.